lib/widgets/content.dart

```diff
@@ -831,8 +839,9 @@ class _InlineContentBuilder {
     }
   }
 
-  InlineSpan _buildStrong(StrongNode node) => _buildNodes(node.nodes,
-    style: weightVariableTextStyle(_context!, wght: 600));
+  InlineSpan _buildStrong(StrongNode node) =>
+    _buildNodes(style: bolderWghtTextStyle(widget.style, by: 200),
+      node.nodes);
```

Comment:
nit: unnecessary churn here, and makes it look different from the similar neighboring methods; can keep `node.nodes` first and the existing formatting